src/lib/components/media/riven/item-manual-scrape.svelte

```
@@ -0,0 +1,822 @@
+		scrapeItem,
+		startManualSession,
+		manualSelect,
+		manualUpdateAttributes,
+		completeManualSession,
+		abortManualSession,
+		parseTorrentTitles,
+        autoScrapeItem,
+        getSettings,
+		type Stream,
+		type StartSessionResponse,
+		type DebridFile,
```

ESLint

[error] 'DebridFile' is defined but never used.

(@typescript-eslint/no-unused-vars)